lib/cloudinary.rb

```diff
@@ -140,18 +128,34 @@ def self.app_root
   end
 
   private
-  
+
+  def self.config_from_env
+    # Heroku support
+    if ENV["CLOUDINARY_CLOUD_NAME"]
+      config_keys = ENV.keys.select! { |key| key.start_with? "CLOUDINARY_" }
+      config_keys -= ["CLOUDINARY_URL"] # handled separately
+      config_keys.each do |full_key|
+        conf_key = full_key["CLOUDINARY_".length..-1].downcase # convert "CLOUDINARY_CONFIG_NAME" to "config_name"
+        conf_val = ENV[full_key]
+        conf_val = conf_val.to_s == 'true' if %w[true false].include?(conf_val.to_s) # cast relevant boolean values
```

Comment:
ENV values should always be strings, so I don't think `to_s` is needed here. It was previously used to deal with a missing ENV var, but those aren't going to happen with the new code.